### ISSUE TITLE
Permission error for accessing vbox-kernel-service

### ISSUE THREAD

**Issue by meolix** [manjaro/release-plan#14]
Testet both  KDE and XFCE Manjaro 15.09-rc1 within Virtualbox
running on Manjaro-Xfce.8.13 64bit kernel 4.1. and 4.2
Issue 1: Permssion error for accessing vbox-kernel-service!
The permission for device /dev/vboxuser is 0600 owend by root.
which resulted in error accessing the vbox-kernel services.
Despite the proper udev-rules 60-vboxguest.rules 0666 for
the device /dev/vboxusers the udev rules is not properly applied.

Fix for issue 1: 
Copying the /usr/lib/udev/rules.d/60-vboxguest.rules 
to /etc/udev/rules.d/60-vboxguest.rules should 
in theory do not change the result of the applied udev-rules,
as both rules have the same name and even the same content.
But building with buildiso a new ISO with the rules copied to
[overlay-livecd]/etc/udev/rules.d/60-vboxguest.rules.
have fixed the above issue (for me).

**Comment by philmmanjaro** [comment]
Well, I'm always testing all images in virtualbox. On my end it works. I need more info why those 60-vboxguest.rules create some issues for you.
```
[manjaro@manjaro ~]$ ls -ltr /dev/vboxuser
crw-rw-rw- 1 root root 10, 56 Sep  3 22:23 /dev/vboxuser
```

**Comment by meolix** [comment]
I'm running latest majaro-xfce 64bit as host with everything updated . I'd already build a couple of iso using buildiso, and some day around the new virtualbox 5 update I got the errror messages in form of 3 to 4 popup within virutalbox fro all newly build isos stating something like erro access to vboxkernel-service. the error show also for all othe systemd-iso's published (e.g. fluxbox, lxqt ...) which I'm also testing. Digging into the issue I discovverd the permssion issue:
[manjaro@manjaro ~]$ ls -l /dev/vboxuser
crw------- 1 root root 10, 56 Sep  3 22:38 /dev/vboxuser

But if I'm the only one who is suffering from this issue, then we might forget this.
I do have additionally the virtualbox extension pack Oracle_VM_VirtualBox_Extension_Pack-5.0.2-102096.vbox-extpack installed. 
I've also run the same ISO in virtuabox on LinuxMint on Virtualbox 5.02 / plus Extension Pack
withthe same Error message on the same computer . So myt assumption was theats more a generic issue. I'll go and will boot up another computer with Ubuntu/Mint /Virtuabox 4.xx, as I might have not yet upgrade to latest Virtualbox on this computer!

**Comment by philmmanjaro** [comment]
Did you tested **15.09-rc1** install medias yet. Those I tested yesterday. However, if the quirk is needed, we can add it for sure.

**Comment by meolix** [comment]
Yes, I've tested the 15.09-rc1 install medias -  both kde and xfce within Vbox 5.01. On my main box having a couple of different OS and flavours I'v tested this. I've got the same VBOXClient -error message an all Vbox hosts (ubuntu, Mint, manjaro). I do manual a chmod 666 /dev/vboxusers.
As I mentioned I do got this on the 0-8.13- systemd-iso's too. But I do not got the error on any openrc-iso's!  Now I'll do fresh install on a spare partition - not in Vbox - on the main box itself, and will run within this insatlled 15.09-rc1 the 15.09-rc1-iso within vbox. But I guess I'll get the VBOXClient -error message. The strange thing about it is that with placing/copying the 60-vbuser.rule under /etc/udev I dont get the error message and the permission /dev/vboxuser is as designed ( ok tested only on iso's I made with buildiso myself!). My gut feeling tells me that this might be a timing issue for the rules applied by systemd-udev. I'll let you know. Any other observations of course too. BTW thanks for the great expierience with manjaro. I'm deeply impressed!

**Comment by meolix** [comment]
Some further comments on manajro1509rc1:
- re: Permission issue on /dev/vboxuser with virtualbox
finaly I managed to install mjro1509 xfce on a spare
partition on my uefi box. After installing virtualbox
I have booted from the ISO within this fresh installed vbox.
So I can confirm I do got the same vbvox-kernel access errors due to missing permssions /dev/vboxuser withtin the newly installed virtualbox on manjaro1509.

- Impressions on my UEFI-installation:
I've tried UEFI-installation of mjro1509rc1-xfce from USB-stick and from dvd.
The first issue was the booting itself either from usb or dvd. I do got stuck at the grub-menu as soon as I touched the keyboard in order to select the non-free boot option (for nvidia card). So I finally have not touched the keyboard until manjaro booted.
The calamares-installer did not even started or did not finsh to show up. My box has quite a lot of partitons about 21 on one hd-drive an 6 partitonson on a ssd-drive - might be that was to heavy for calamares.

OK now I tried thus-installler. At least this went through - nearly. Only at the end I got warning that the grub-installtion somehow went wrong. Not only the grub-installer went wrong- I could see a kernel vmlinuz image but no initramfs.

Finally I've re-done and finished installation using CLI-installer. This went nearly fine. But the installer seemd also to have issues with the number of partitons as I could not manually select my last sda21-swap partiton. Which later gave me some complains about missing resume-devices. CLI-installer also warned me about some potential issues creating a uefi-menu entry.

OK now booting back and selecting manjaro uefi-menu entrie I was back in my existing manjaro-0.8.13. Regnerating the grub-menu and I could now boot from the grub menu into manjaro-1509rc1. After this hard uefi-install way I got it finally: The goog news: It is now on my ssd-drive on boots in 3-4 seconds and is damn fast.

**Comment by philmmanjaro** [comment]
So this issue is still up?

**Comment by meolix** [comment]
Hi Phil!
The issue is now fixed!  I've just tested this with 15.09-rc3 isos's 
(kde, xfce, i686 , x86_64 ) within Virtuabox 5.0.2 on manjaro-host xfce-15.09-rc3 with kernel 4.2
The Virtualbox-Kernel permission errors do not show up anymore and /dev/vboxuser has permissions set as defined by the 60-vboxguest-udev-rules.
Big thanks from Hamburg/Germany - deeply impressed!
